httpapi/httpapi.go

```diff
@@ -80,14 +80,8 @@ func recordFetchHandler(hy *hydra.Hydra) func(w http.ResponseWriter, r *http.Req
 		for peerAddrInfo := range hy.Sybils[0].Routing.FindProvidersAsync(ctx, cid, nProviders) {
 			// fmt.Printf("Got one provider %s\n", peerAddrInfo.String())
 			// Store the Provider locally
-			hy.Sybils[0].Routing.ProviderManager.AddProvider(ctx, cid.Bytes(), peerAddrInfo.ID)
-			// AddProvider doesn't automatically flush it to the datastore.
-			// A GetProviders helps wash it down. Useful for monitoring number of records stored
-			res := hy.Sybils[0].Routing.ProviderManager.GetProviders(ctx, cid.Bytes())
-			fmt.Printf("Stored new Provider Record for CID: %s with providers: %s\n", cidStr, res)
```

Comment:
No need anymore?

Reply:
Yes sorry! I forgot to ask - as far as I can tell AddProvider does flush to disk. I removed this and things still seem to be working (at least tests are passing). Do you remember what the issue was?

Reply:
It was failing for me without it. Maybe it changed with newest deps.